### ISSUE TITLE
Site index

### ISSUE THREAD

**Issue by clgiebink** [clgiebink/UT_FVS#15]
Site index (SICOND) need to be aligned to the same base age (SIBASE) and site species (SISP). What is the best way to do this?

**Comment by clgiebink** [comment]
From John D. Shaw:

The best way to approach this would be to use PLT_CN to attach to the SITETREE table, and grab trees (species-specific) that are associated with those plots. SITREE should be populated so you can use those values directly. If more than one site tree is present, you can average them for a condition. Fortunately, FIA and FVS use the same equation for DF in Utah (Brickell 1970; a.k.a. PSME11 in our library).

**Comment by clgiebink** [comment]
Matching SITREE worked for the validation data set. I averaged SITREE when more than one tree was present. I wasn't able to get a match for any for any trees in the calibration data set (tree ring, periodic) where SPCD ≠ SISP using the SITETREE table. I decided to drop those trees when fitting the models for now. In doing so, I lost 36 DF (136 -> 100), 14 PP (87 -> 73), and 17 ES (95 -> 78) trees. In the future, I could calculate site index from height and age. Does FIA and FVS use the same calculations for ponderosa pine and Engelmann spruce too?
